apps/L1G/restaurant/restaurant-frontend/specs/(admin)/table/components/CreateTableModal.spec.tsx

```
@@ -0,0 +1,143 @@
+import { CreateTableModal } from '@/components/table/CreateTableModal';
+import { CreateTableDocument } from '@/generated';
+import { ApolloError } from '@apollo/client';
```

ESLint

[error] 'ApolloError' is defined but never used.

(no-unused-vars)